OPHD/States/MapViewStateTurn.cpp

```diff
@@ -723,7 +732,15 @@ void MapViewState::nextTurn()
 	updateMaintenance();
 	updateCommercial();
 	updateBiowasteRecycling();
-	updateMorale();
+
+	// Morale will not change until at least n turns from landing, depending on difficulty
+	if (turnsSinceLanding > gracePeriod[mDifficulty])
```

Comment:
This is separate from the crime check above so as not to change the sequential order of crime update, population update, morale update.